src/test/java/net/logstash/logback/encoder/LogstashEncoderTest.java

```diff
@@ -192,213 +187,212 @@ public JsonGenerator decorate(JsonGenerator generator) {
                 + "  level_value : 40000%n"
                 + "}%n"));
     }
-    
+
 
     @Test
     public void loggerNameIsShortenedProperly() throws Exception {
         final long timestamp = System.currentTimeMillis();
         final int length = 36;
-        final String shortenedLoggerName = new TargetLengthBasedClassNameAbbreviator(length).abbreviate(FastDateFormat.class.getCanonicalName());
+        final String shortenedLoggerName = new TargetLengthBasedClassNameAbbreviator(length).abbreviate(Instant.class.getCanonicalName());
 
         ILoggingEvent event = mockBasicILoggingEvent(Level.ERROR);
-        when(event.getLoggerName()).thenReturn(FastDateFormat.class.getCanonicalName());
+        when(event.getLoggerName()).thenReturn(Instant.class.getCanonicalName());
 
         when(event.getTimeStamp()).thenReturn(timestamp);
         encoder.setFieldNames(new ShortenedFieldNames());
         encoder.setShortenedLoggerNameLength(length);
         encoder.start();
         byte[] encoded = encoder.encode(event);
-        
+
         JsonNode node = MAPPER.readTree(encoded);
 
-        assertThat(node.get("@timestamp").textValue()).isEqualTo(FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format
-                (timestamp));
+        assertThat(node.get("@timestamp").textValue()).isEqualTo(String.valueOf(timestamp));
```

Comment:
Same comment as above. Why is the expected value here a long value converted to a string?

Reply:
I missunderstood the StringTimestampWriter, as a result I've done some wrong changes here